src/app/redux/SagaShared.js

```diff
@@ -85,5 +88,5 @@ function* saveUserPreferences({payload}) {
 }
 
 function* watchUserSettingsUpdates() {
-    yield* takeLatest(['SET_USER_PREFERENCES', 'TOGGLE_NIGHTMODE', 'TOGGLE_BLOGMODE'], saveUserPreferences);
+    yield* takeLatest(['app/SET_USER_PREFERENCES', 'app/TOGGLE_NIGHTMODE', 'app/TOGGLE_BLOGMODE'], saveUserPreferences);
```

Comment:
Wondering if these strings should be constants.

Reply:
we could export the action types & make them available for the sagas. also it'd be good to explicitly note in the reducers when an action has a saga, like has been done here https://github.com/steemit/condenser/blob/master/src/app/redux/User.js#L92

i'd rather make these changes in a future PR though

Reply:
actually i'm doing it now